packages/build-utils/test/unit.convert-runtime-to-plugin.test.ts

```diff
@@ -0,0 +1,130 @@
+import { join } from 'path';
+import { readFile, remove, existsSync } from 'fs-extra';
+import { BuildOptions, createLambda } from '../src';
+import { convertRuntimeToPlugin } from '../src/convert-runtime-to-plugin';
+
+const workPath = join(__dirname, 'walk', 'python-api');
+
+describe('convert-runtime-to-plugin', () => {
+  afterEach(async () => {
+    await remove(join(workPath, '.output'));
+  });
+
+  it('should create correct fileystem for python', async () => {
+    const buildRuntime = async (opts: BuildOptions) => {
+      const lambda = await createLambda({
+        files: opts.files,
+        handler: 'index.handler',
+        runtime: 'python3.9',
+      });
+      return { output: lambda };
+    };
+
+    const build = await convertRuntimeToPlugin(buildRuntime, '.py');
+    await build({ workPath });
+    const apiDir = join(workPath, '.output', 'server', 'pages', 'api');
+
+    const indexJson = await readFile(join(apiDir, 'index.py.nft.json'), 'utf8');
+    expect(JSON.parse(indexJson)).toMatchObject({
+      version: 1,
+      files: [
+        {
+          input: '../../../../inputs/runtime-temp/api/index.py',
```

Comment:
So this is the entrypoint file right? It already gets copied into the output lambda at `.output/server/pages/api/index.py`. So are we copying it again here because the default output path is not suitable?

Reply:
This is the part I'm not sure about.

I didn't create a `.output/server/pages/api/index.py` file but I guess I could?

Reply:
The FS API already does that is what I'm saying. If you have `.output/server/pages/api/index.py` as the entrypoint in the output files, the FS API will include that entrypoint file at `.output/server/pages/api/index.py` _within the lambda_. This is because it's what Next.js expects, but IMO not a good default for serverless functions (where I'd expect the same path as the entrypoint source file i.e. `api/index.py`, not the output path).

So as a net result I believe the `index.py` entrypoint file will be included in the Lambda _twice_ (once at `.output/server/pages/api/index.py` as done implicitly by FS API), and again here in the nft file where it will be placed at `api/index.py`.

Reply:
I replaced the need for nft.json in 1f03667

Reply:
@TooTallNate I spoke to Andy and learned that we need to continue using nft.json so I added it back in c5599d9 along with the proper entrypoint file `.output/server/pages/api/index.py`